### ISSUE TITLE
Screen Reader accessibility

### ISSUE THREAD

**Issue by DarkMiso** [harvesthq/chosen#990]
Hi there,

I was trying to make some accessibility changes to the component, was able to reach a better working high contrast black version, but i am having problems with screen readers.

I was able to make it so, that screen reader capable of telling you the name(title) of the selection field, but no matter where i set alt or title attribute(or both), every option is read as "blank", do any of you have an idea, how could i set those attributes, so that screen readers capable telling the user, which option is currently selected during navigation with key arrows?

Best regards,
Mihaly
